README.md

```diff
@@ -20,12 +20,6 @@ them to CCFv3.
 
 ## Installation
 ### Python Version and Environment
-Note that due to some of our dependencies we're currently limited to python
-version `3.7`. Please make sure you set up a virtual environment with that
-version before trying to install this library. If you're unsure how to do that
-please have a look at [conda](https://docs.conda.io) or
-[pyenv](https://github.com/pyenv/pyenv).
-
 If you are part of the Blue Brain Project and are working on the BB5 you can
 find the correct python version in the archive modules between `archive/2020-02`
 and `archive/2020-12` (inclusive). Here's an example of a set of commands
```

Comment:
I would update the BB5 information too. Here are the archive modules for the python versions we support:
* python3.7: `archive/2020-02` - `archive/2020-12`
* python3.8: `archive/2021-01` - `archive/2021-12`
* python3.9: `archive/2022-01` - `unstable`

Reply:
Thanks for the info. I updated it in 5cf6477.